### ISSUE TITLE
adding support for Ws2801 LEDs

### ISSUE THREAD

**Issue by Rockiii87** [arendst/Sonoff-Tasmota#2272]
Hi Theo. 
I'm using Tasmota on several Sonoffs, but i need it also on my NodeMCU to light up a stripe of WS2801 and get input from a DHT11 Sensor at the same time.

Would it be possible to get it in the code for the NodeMCU's?

**Comment by Jason2866** [comment]
Why does it not work?
Tasmota works the same way as on a sonoff device.

**Comment by Rockiii87** [comment]
yes, but not with the WS2801, as i think. Because the WS2801 is a 4 Pin Stripe.

**Comment by Jason2866** [comment]
Your description is missunderstanding for that case.
Just write in headline feature request for WS201 and nothing else...

**Comment by arendst** [comment]
The WS2801 interface is fundamentally different from WS2812. The WS2801 is an older version led strip than WS2812 so I do not expect much demand for it.

I currently have no plans to implement unless someone comes up with a pull request.

**Comment by Jonn3y** [comment]
Would love to see this being done some time. NeoPixelBus supports the WS2801 since Jan. 2020 (v2.5.6)
https://github.com/Makuna/NeoPixelBus/issues/324

**Comment by Jason2866** [comment]
@Jonn3y  As Theo wrote if someone provides a PR he will probably merge
